<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Diretório Origem</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_diretorio_origem"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget_processado"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>454</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionBuscarDiretorio"/>
    <addaction name="actionSalvar"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuConfigura_es">
    <property name="title">
     <string>Configurações</string>
    </property>
    <addaction name="actionPlano_Amostral"/>
    <addaction name="actionRegras_de_Descarte"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionSobre"/>
    <addaction name="actionDesenvolvedor"/>
   </widget>
   <widget class="QMenu" name="menuConstruir">
    <property name="title">
     <string>Construir</string>
    </property>
    <addaction name="actionProcessar"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuConstruir"/>
   <addaction name="menuConfigura_es"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBuscarDiretorio">
   <property name="text">
    <string>Buscar Diretório</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Exportar Dados</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionPlano_Amostral">
   <property name="text">
    <string>Plano Amostral</string>
   </property>
  </action>
  <action name="actionRegras_de_Descarte">
   <property name="text">
    <string>Regras de Descarte</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionDesenvolvedor">
   <property name="text">
    <string>Desenvolvedor</string>
   </property>
  </action>
  <action name="actionProcessar">
   <property name="text">
    <string>Processar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
